### PR TITLE
Version and command-line

### PR DESCRIPTION
**Pull request by leafyoung**
DEPENDING on PR in runner-api.
https://github.com/credmark/credmark-model-runner-api/pull/39

- add checking for latest version from gateway

### CODE REVIEW

File: credmark/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '0.8.22'
```
Comment (iwyrkore) [inline credmark/__init__.py:1]: This __init__.py file cannot be here. credmark folder is a namespace.

Comment (iwyrkore) [reply credmark/__init__.py:1]: The version is related to the cmf folder anyhow.

File: credmark/__main__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from .cmf import credmark_dev
```
Comment (iwyrkore) [inline credmark/__main__.py:1]: This file should not be here.

File: credmark/cmf/engine/context.py [hunk 3]
```diff
@@ -126,6 +127,14 @@ def create_context_and_run_model(cls,  # pylint: disable=too-many-arguments,too-
                 'Using local models (requested + dev models): '
                 f'{cls.use_local_models_slugs}')
 
+            remote_version = cls.get_cmf_version(api, chain_id)
```
Comment (iwyrkore) [inline credmark/cmf/engine/context.py:130]: We should move this code somewhere else as a separate function.

File: credmark/__init__.py (hunk 1 above)
Comment (leafyoung) [reply credmark/__init__.py]: You could have a try of https://pypi.org/project/versioneer/
https://github.com/python-versioneer/python-versioneer

File: credmark/__main__.py (hunk 2 above)
Comment (leafyoung) [reply credmark/__main__.py]: moved to credmark/cmf/

**Pull request merged**